MVP_Clicker/Assets/Project/Scripts/Game/Base/GameViews/GameViews.cs

```diff
@@ -1,17 +1,26 @@
+using Project.Scripts.Core.LoadResourcesService;
 using Project.Scripts.Core.ViewCreate;
 using Project.Scripts.Game.Areas.Camera.View;
 using Project.Scripts.Game.Areas.MainMenu.View;
-using UnityEngine;
+using Unity.VisualScripting;
+using Object = UnityEngine.Object;
 
 namespace Project.Scripts.Game.Base.GameViews
 {
-    public class GameViews : MonoBehaviour, IGameViews
+    public class GameViews : IGameViews
     {
-        [SerializeField] private CameraView _cameraPrefab;
-        [SerializeField] private MainMenuView _mainMenuPrefab;
+        private readonly CameraView _cameraPrefab;
+        private readonly MainMenuView _mainMenuPrefab;
 
         public IViewCreator<ICameraView> Camera => new ViewCreator<CameraView>(_cameraPrefab);
         public IViewCreator<IMainMenuView> MainMenu => new ViewCreator<MainMenuView>(_mainMenuPrefab);
 
+        public GameViews(ILoadResourcesService loadResourcesService)
+        {
+            var cameraObject = loadResourcesService.Load<Object>("Assets/Project/Prefabs/MainCamera.prefab");
+            _cameraPrefab = cameraObject.ConvertTo<CameraView>();
+            var mainMenuObject = loadResourcesService.Load<Object>("Assets/Project/Prefabs/MainMenu.prefab");
+            _mainMenuPrefab = mainMenuObject.ConvertTo<MainMenuView>();
```

Comment:
You are loading assets in constructor. I think is more good to load it in another method. I mean you can make some IInitializable interface that will has a method `Task InitializeAsync()` and will call in async Start method of unity. That will not lag the game